### PR TITLE
Fixes Atheism

### PR DESCRIPTION
**Pull request by Thunder12345**
## About The Pull Request

Makes suiciding with the atheist's fedora have you trying to get further from god, rather than closer to god

## Why It's Good For The Game

Improves representation of atheists in our code by not incorrectly associating them with an interest in getting closer to god.

## Changelog

:cl:
fix: Atheists will no longer mistakenly attempt to get closer to a god they do not believe in when committing suicide with their fedora
/:cl:

### CODE REVIEW

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -532,6 +532,10 @@
 	attack_verb_simple = list("enlighten", "redpill")
 	menu_description = "A sharp fedora dealing a very high amount of throw damage, but none of melee. Fits in pockets. Can be worn on the head, obviously."
 
+/obj/item/nullrod/fedora/suicide_act(mob/user)
+	user.visible_message(span_suicide("[user] is killing [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to get further from god!"))
```
Comment (Mothblocks) [inline code/game/objects/items/holy_weapons.dm:536]:
```suggestion
	user.visible_message(span_suicide("[user] is killing [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to get further from God!"))
```

Comment (ATH1909) [reply code/game/objects/items/holy_weapons.dm:536]: the normal null rod suicide also doesn't capitalize "god"

Comment (Mothblocks) [reply code/game/objects/items/holy_weapons.dm:536]: eh

Comment (Farquaar) [reply code/game/objects/items/holy_weapons.dm:536]: "God" is a proper noun. As in, "He believes in God." or "He prayed to God."
"god" is not a proper noun.  As in, "He believes in a god." or "He prayed to some god or another."
As there is no article in this sentence, I would infer that "God" is being used as a proper noun and would require a capital.

Comment (Ryll-Ryll) [reply code/game/objects/items/holy_weapons.dm:536]: Wouldn't it be better to just use whatever deity name the chap set and sidestep the whole issue?

Comment (tralezab) [reply code/game/objects/items/holy_weapons.dm:536]: Yeah but if someones running the atheism gimmick they'd be getting further from anti-god or whatever

**Comment by wesoda25** [comment]
It's funny 👎

File: code/game/objects/items/holy_weapons.dm (hunk 1 above)
Comment (darkmanx37) [reply code/game/objects/items/holy_weapons.dm:536]: I believe "gods" would be correct, God by big g implies monotheism, while atheist doesn't believe in any god, from monotheist and politheist religions alike. Also, "god" is a proper noun while talking about politheistic religions, or many at once at least. Zeus was a god of thunder, christian god shares some traits with Aztec gods, etc. Not to get into the fact that capitalizing type of creature (god) of something that has a name (Yahwe, in christianity's case) is only to show the respect to it, which atheist doesn't really have reason to have, but whatever.

Comment (tralezab) [reply code/game/objects/items/holy_weapons.dm:536]: image

**Pull request merged**